dist/aurora.js

```diff
@@ -53,6 +53,41 @@ function getNuxtVersion() {
   return window['__unctx__']?.get('nuxt-app')?.use()?.versions?.nuxt;
 }
 
+// Detects Next.js version
+function getNextVersion() {
+  return window.next.version;
```

Comment:
I think `runSafely` will catch this but we should make sure to gracefully return something if there is no `window.next`

```suggestion
  return window.next?.version;
```